crates/net/downloaders/src/bitfinity_evm_client.rs

```diff
@@ -46,6 +49,19 @@ pub struct RpcClientConfig {
     pub max_retries: u32,
     /// Delay between retries
     pub retry_delay: Duration,
+    /// Block validation client
+    pub evm_block_validator: Option<BitfinityBlockValidator<IcAgentClient, Arc<DatabaseEnv>>>,
```

Comment:
It makes me sad to see a validator be a part of a config... Maybe we could refactor it somehow?..